recruitment/positions/android_chapter_lead.md

```diff
@@ -0,0 +1,86 @@
+<p align="center">
+<img src="../../logo.png">
+</p>
+
+Android Chapter Lead
+==================================
+
+Babylon aims to hire only the most qualified people available for the 
+job at any given time. That doesn't mean that a candidate must have 
+several academic degrees or have years and years of experience. 
+We value potential equally as much as knowledge and experience. 
+Above all, we value one's ability to think independently and to engage.
+
+Additionally, we want to hire people who are reasonable, professional, 
+and fun to work with. After all, they are people we will spend a great 
+deal of time interacting with on a daily basis.
+
+We are continually improving how we work remotely as a company and aim 
+to be as flexible as possible by equally accepting applications from 
+candidates who want to work on-site at our office in London, or who 
+ant to work remotely.
+
+At Babylon we also believe in/champion diversity and welcome everyone 
+from everywhere. Currently in the Android team you'll find people from 
+Greece, Poland, UK, India, Spain, Portugal, France, Lithuania, Sudan, 
+Italy and Hungary, can you add to it?
```

Comment:
maybe `and Hungary. Can you add to it?` then maybe an emoji if possible!

Reply:
I added Hungary, it's there 👓

Reply:
replaced the list of countries with a line of flags